{"trial_id": "01", "hyperparameters": {"space": [{"class_name": "Int", "config": {"name": "units_2", "default": null, "conditions": [], "min_value": 16, "max_value": 64, "step": 16, "sampling": "linear"}}, {"class_name": "Int", "config": {"name": "units_1", "default": null, "conditions": [], "min_value": 32, "max_value": 128, "step": 32, "sampling": "linear"}}, {"class_name": "Float", "config": {"name": "learning_rate", "default": 0.0001, "conditions": [], "min_value": 0.0001, "max_value": 0.01, "step": null, "sampling": "log"}}], "values": {"units_2": 16, "units_1": 32, "learning_rate": 0.0025926424851734835}}, "metrics": {"metrics": {}}, "score": null, "best_step": 0, "status": "FAILED", "message": "Traceback (most recent call last):\n  File \"C:\\Users\\Charl\\AppData\\Local\\Programs\\Python\\Python39\\lib\\site-packages\\keras_tuner\\src\\engine\\base_tuner.py\", line 273, in _try_run_and_update_trial\n    self._run_and_update_trial(trial, *fit_args, **fit_kwargs)\n  File \"C:\\Users\\Charl\\AppData\\Local\\Programs\\Python\\Python39\\lib\\site-packages\\keras_tuner\\src\\engine\\base_tuner.py\", line 238, in _run_and_update_trial\n    results = self.run_trial(trial, *fit_args, **fit_kwargs)\n  File \"C:\\Users\\Charl\\AppData\\Local\\Programs\\Python\\Python39\\lib\\site-packages\\keras_tuner\\src\\engine\\tuner.py\", line 306, in run_trial\n    for execution in range(self.executions_per_trial):\nTypeError: 'float' object cannot be interpreted as an integer\n"}